io.github.face_hh.Napture.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "io.github.face_hh.Napture",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "napture",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--own-name=org.bussin.napture"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/napture/napture/cargo"
+        }
+    },
+    "modules": [
+        {
+            "name": "lua",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make all install",
+                "install -Dm644 lua.pc /app/lib/pkgconfig/lua.pc"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.lua.org/ftp/lua-5.4.6.tar.gz",
```

Comment:
This should use shared-modules https://github.com/flathub/shared-modules/blob/master/lua5.4/lua-5.4.json

If you need 5.4.6 I can update it there.

Reply:
To be honest, I didn't even know it existed, I'll incorporate it into the manifest once I get to it.